clang/test/SemaSYCL/reqd-work-group-size-device.cpp

```diff
@@ -62,13 +55,6 @@ class Functor16x16x16 {
   [[sycl::reqd_work_group_size(16, 16, 16)]] void operator()() const {}
 };
 
-class Functor8 { // expected-error {{conflicting attributes applied to a SYCL kernel}}
-public:
-  [[sycl::reqd_work_group_size(1, 1, 8)]] void operator()() const { // expected-note {{conflicting attribute is here}}
-    f4x1x1();
-  }
-};
-
 class Functor {
```

Comment:
I would remove this and keep Functor8 and see that Funtor8 does not cause diagnostics since attributes are not propagated

Reply:
>>I would remove this and keep Functor8 and see that Funtor8 does not cause diagnostics since attributes are not propagated

Agree with @elizabethandrews